src/Infrastructure/Captcha/GoogleRecaptchaV3Service.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Microsoft.Extensions.Configuration;
+using System.Runtime.Serialization.Json;
+using System.Web;
+
+namespace FSH.WebApi.Infrastructure.Captcha;
+
+public class GoogleRecaptchaV3Service : ICaptchaService
+{
+    public HttpClient _httpClient { get; set; }
+
+    public string RequestPath { get; set; }
+    public string RequestSecret { get; set; }
+    public string RequestResponse { get; set; }
+    public string RequestRemoteip { get; set; }
+
+    public GResponseModel Response { get; set; }
+
+    public HttpRequestException HttpReqException { get; set; }
+
+    public Exception GeneralException { get; set; }
```

Comment:
Why is this all public here? and all with public setters? Doesn't make much sense...

This whole captchaservice api surface can be reduced to basically one method. nice and easy... please try to keep it simple.